be/src/olap/rowset/unique_rowset_id_generator.h

```diff
@@ -28,7 +28,10 @@ namespace doris {
 class UniqueRowsetIdGenerator : public RowsetIdGenerator {
 public:
     UniqueRowsetIdGenerator(const UniqueId& backend_uid);
-    ~UniqueRowsetIdGenerator();
+    ~UniqueRowsetIdGenerator() override;
+
+    UniqueRowsetIdGenerator(const UniqueRowsetIdGenerator&) = delete;
+    UniqueRowsetIdGenerator operator=(const UniqueRowsetIdGenerator&) = delete;
```

Comment:
```suggestion
    UniqueRowsetIdGenerator& operator=(const UniqueRowsetIdGenerator&) = delete;
```
See [https://en.cppreference.com/w/cpp/language/copy_assignment](https://en.cppreference.com/w/cpp/language/copy_assignment)